corehq/motech/management/commands/create_repeat_records.py

```diff
@@ -18,17 +18,25 @@ class Command(BaseCommand):
     help = "Create repeat records for cases and forms"
 
     def add_arguments(self, parser):
-        parser.add_argument('file_path', help='Path to the list of IDs. One ID per line.')
+        parser.add_argument('-d', '--doc-id', nargs='*',
+                            help='Create records for the given doc ID(s)')
+        parser.add_argument('-f', '--file-path',
```

Comment:
I assume that file-path is available as file_path later?

Reply:
Yes, you're right. `argparse` makes the '--file-path' option available as `options['file_path']`.